packager/media/formats/mp4/box_definitions.h

```diff
@@ -382,6 +389,13 @@ struct AudioSampleEntry : Box {
   AC4Specific dac4;
   OpusSpecific dops;
   FlacSpecific dfla;
+  MHAConfiguration mhac;
+
+  //23008-3 decoder specific config
+  CodecConfiguration codec_configuration;
+
+  // Returns the box type of codec configuration box from audio format.
+  FourCC GetCodecConfigurationBoxType(FourCC format) const;
```

Comment:
MPEG-H decoders need a codec_configuration values, similar to video codecs. So I added a codec_configuration box to allow for the holding and parsing of the configuration_stream.

If there is a better way to implement, please suggest.

Reply:
Shouldn't it be handled in `MHAConfiguration mhac` already? Isn't the mhac the codec configuration for MPEG-H? 

I understand that it is a bit confusing, but the approach for audio and video are slightly different. For audio, we are listing all the possible audio decoder configurations while for video, we are using the same CodecConfiguration structure. The code should really be cleaned up.

I think we can just remove line 393 to 398. The functionality has already been implemented in `mhac`. Let me know if I misunderstood your intention and sorry for the confusion.

Reply:
Removed 393 - 398